cli/Sources/TuistSupport/Xcode/Xcode.swift

```diff
@@ -3,12 +3,17 @@ import Foundation
 import Path
 
 public enum XcodeError: FatalError, Equatable {
```

Comment:
Since you are touching this symbol, would you mind making it conform `LocalizedError` instead of `FatalError` and use `errorDescription` instead of `description`?